test/gtest/ucs/test_config.cc

```diff
@@ -10,6 +10,8 @@
 extern "C" {
 #include <ucs/config/parser.h>
 #include <ucs/time/time.h>
+#undef UCX_CONF_FILE
+#define UCX_CONF_FILE "ucs/ucx.conf"
```

Comment:
why needed? it wouldn't affect parser.c anyway right?

Reply:
For the gtest. Do you prefer to remove it?

Reply:
i mean, why need undef+define?
can we use static const char *test_config_file="..."?